Checks failing with

```
FAILED test_app/tests/jwt_consumer/common/test_auth.py::TestJWTCommonAuth::test_process_rbac_permissions_cache_scenarios[True-True-True-False-False-False] - AttributeError: <module 'ansible_base.jwt_consumer.common.auth' from '/home/runner/work/django-ansible-base/django-ansible-base/ansible_base/jwt_consumer/common/auth.py'> does not have the attribute 'get_user_claims'
FAILED test_app/tests/jwt_consumer/common/test_auth.py::TestJWTCommonAuth::test_process_rbac_permissions_cache_scenarios[False-True-True-True-False-False] - AttributeError: <module 'ansible_base.jwt_consumer.common.auth' from '/home/runner/work/django-ansible-base/django-ansible-base/ansible_base/jwt_consumer/common/auth.py'> does not have the attribute 'get_user_claims'
FAILED test_app/tests/jwt_consumer/common/test_auth.py::TestJWTCommonAuth::test_process_rbac_permissions_cache_scenarios[False-False-True-True-True-True] - AttributeError: <module 'ansible_base.jwt_consumer.common.auth' from '/home/runner/work/django-ansible-base/django-ansible-base/ansible_base/jwt_consumer/common/auth.py'> does not have the attribute 'get_user_claims'
FAILED test_app/tests/jwt_consumer/common/test_auth.py::TestJWTCommonAuth::test_process_rbac_permissions_cache_scenarios[False-False-False-False-True-False] - AttributeError: <module 'ansible_base.jwt_consumer.common.auth' from '/home/runner/work/django-ansible-base/django-ansible-base/ansible_base/jwt_consumer/common/auth.py'> does not have the attribute 'get_user_claims'
```

At quick glance - almost certainly due to assumptions in test mocking. This should be relatively easy to resolve, perhaps by moving the mock to ansible_base/rbac.

--- a/ansible_base/jwt_consumer/common/auth.py
+++ b/ansible_base/jwt_consumer/common/auth.py
@@ -16,7 +16,6 @@ from ansible_base.jwt_consumer.common.exceptions import HTTP_498_INVALID_TOKEN, 
 from ansible_base.lib.logging.runtime import log_excess_runtime
 from ansible_base.lib.utils.auth import get_user_by_ansible_id
 from ansible_base.lib.utils.translations import translatableConditionally as _
-from ansible_base.rbac.claims import get_claims_hash, get_user_claims, get_user_claims_hashable_form, save_user_claims
 from ansible_base.resource_registry.models import Resource, ResourceType
 from ansible_base.resource_registry.rest_client import get_resource_server_client
 from ansible_base.resource_registry.signals.handlers import no_reverse_sync
@@ -230,6 +229,8 @@ class JWTCommonAuth:
         """
         Process RBAC permissions using claims hash logic
         """
+        from ansible_base.rbac.claims import get_claims_hash, get_user_claims, get_user_claims_hashable_form, save_user_claims
+
         if self.token is None or self.user is None:
             logger.error("Unable to process rbac permissions because user or token is not defined")
             return

--- a/test_app/tests/jwt_consumer/common/test_auth.py
+++ b/test_app/tests/jwt_consumer/common/test_auth.py
@@ -480,11 +480,11 @@ class TestJWTCommonAuth:
         with (
             mock.patch.object(authentication.cache, 'get_cached_claims_hash') as mock_get_cache,
             mock.patch.object(authentication.cache, 'cache_claims_hash') as mock_set_cache,
-            mock.patch('ansible_base.jwt_consumer.common.auth.get_user_claims') as mock_get_claims,
-            mock.patch('ansible_base.jwt_consumer.common.auth.get_user_claims_hashable_form') as mock_get_hashable,
-            mock.patch('ansible_base.jwt_consumer.common.auth.get_claims_hash') as mock_get_hash,
+            mock.patch('ansible_base.rbac.claims.get_user_claims') as mock_get_claims,
+            mock.patch('ansible_base.rbac.claims.get_user_claims_hashable_form') as mock_get_hashable,
+            mock.patch('ansible_base.rbac.claims.get_claims_hash') as mock_get_hash,
             mock.patch.object(authentication, '_fetch_jwt_claims_from_gateway') as mock_gateway,
-            mock.patch('ansible_base.jwt_consumer.common.auth.save_user_claims') as mock_apply,
+            mock.patch('ansible_base.rbac.claims.save_user_claims') as mock_apply,
         ):
 
             # Setup mocks

